### PR TITLE
Tugas SIOT Fork Calvin X SIJA

### PR DESCRIPTION
**Pull request by Zepicxs**

### CODE REVIEW

File: initiates/Calvin_XSIJA_IfStatement.ino
Comment (SyaddadRaihanPutra) [inline initiates/Calvin_XSIJA_IfStatement.ino:1]: Please correct the code again, there is still something wrong. The code doesn't run well 🔥

**Review by SyaddadRaihanPutra — CHANGES_REQUESTED** [review]
Please correct the code again, there is still something wrong. The code doesn't run well 🔥

**Comment by Zepicxs** [comment]
> Please correct the code again, there is still something wrong. The code doesn't run well 🔥

Okeh

**Comment by Zepicxs** [comment]
Where is the error sir? i didnt found any error in my tinker cad and it works properly. smh💀

**Review by SyaddadRaihanPutra — COMMENTED** [review]
Sudah saya cek masih ada kesalahan pada percabangan kondisi nya, saya menguji dengan Wokwi dan TinkerCad

**Comment by Zepicxs** [comment]
> Sudah saya cek masih ada kesalahan pada percabangan kondisi nya, saya menguji dengan Wokwi dan TinkerCad

okey akan saya cek lagi

**Pull request closed without merging**